examples/stress-entities/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Stress Multiple Entities</title>
+    <meta name="description" content="Stress Multiple Entities - A-Frame">
+    <script src="../../dist/aframe.js"></script>
+    <style>
+      body { background-color: black; }
+      .stats {
+        position: fixed;
+        text-align: center;
+        top: 200px;
+        right: calc(50% - 225px);
+        left: calc(50% - 225px);
+        width: 450px;
+        height: 75px;
+        color: white;
+        padding: 20px;
+        font-size: 20pt;
+        font-family: Helvetica;
+        font-weight: medium;
+        line-height: 1.6em;
+        background-color: #EF2D5E;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="stats"></div>
+    <a-scene stats="true">
+      <a-assets>
+        <a-mixin id="cube" geometry="primitive: box; height: 16; width: 16; depth: 16;"
+                 material="color: #167341; roughness: 1.0; metalness: 0.2;"></a-mixin>
+      </a-assets>
+      <a-entity position="0 20 0">
+        <a-entity camera="active:true" rotation="0 0 0" position="0 0 0"></a-entity>
+      </a-entity>
+    </a-scene>
+    <script>
+      var cubeEl;

```

Comment:
Would be cool to encourage "component-driven development", but that's if you want to.